analysis/metrics.py

```diff
@@ -20,37 +20,12 @@
 import math
 
 
-@dataclass
-class CountMetrics:
-    """Stores metrics for the count utility analysis.
-
-  Attributes:
-      count: actual count of contributions per partition.
-      per_partition_error: the amount of error due to per-partition
-      contribution bounding.
-      expected_cross_partition_error: the expected amount of error due to
-      cross-partition contribution bounding.
-      std_cross_partition_error: the standard deviation of the error due to
-      cross-partition contribution bounding.
-      std_noise: the noise standard deviation.
-      noise_kind: the type of noise used.
-
-  s.t. the following holds (where E stands for Expectation):
-  E(count_after_contribution_bounding) = count + E(error)
-  where E(error) = per_partition_error + expected_cross_partition_error
-  """
-    count: int
-    per_partition_error: int
-    expected_cross_partition_error: float
-    std_cross_partition_error: float
-    std_noise: float
-    noise_kind: pipeline_dp.NoiseKind
-
-
 @dataclass
 class SumMetrics:
     """Stores metrics for the sum utility analysis.
 
+  Also used for count & privacy_id_count.
+
```

Comment:
Nit: in  the docstring in this and other dataclasses in this file the spacing is weird. Could you please fix it?
```
class ...
        """" # (4 spaces before """")
        text
        """" 
```